### PR TITLE
Added fit_regressor function documentation

### PR DESCRIPTION
**Pull request by rezam747**
Added fit_regressor function documentation

### CODE REVIEW

File: R/fit_regressor.R [hunk 1]
```diff
@@ -1 +1,14 @@
...
+#'
+#' @examples
+#' data <- read_csv("data/spotifyfeatures.csv")
+#' fit_regressor(data, target_col = 'popularity', categorical_features='genre')
+#' fit_regressor(data, target_col = 'popularity', numeric_feats = ['danceability', 'loudness'], categorical_feats=['genre'], cv=10)
+#' 
+#' df <- data.frame(x = c(1, 2, 4, 6, 8),y = c(3, 6, 12, 18, 24))
+#' fit_regressor(df, target_col = 'y', numeric_feats='x')
+#' fit_regressor(df, target_col = 'y')
 fit_regressor <- function(){}
```
Comment (nd265) [inline R/fit_regressor.R]:
```suggestion
fit_regressor <- function(){}
```
Can you add input variables in this function?

**Review by nd265 — APPROVED** [review]
Please add @param in the doc

**Review by zzhzoe — APPROVED** [review]

**Review by Sanchit120496 — CHANGES_REQUESTED** [review]
Input variable params needed

**Review by nd265 — APPROVED** [review]

**Review by nd265 — APPROVED** [review]

**Review by nd265 — CHANGES_REQUESTED** [review]
Input @params required

**Review by nd265 — APPROVED** [review]

**Review by Sanchit120496 — APPROVED** [review]

**Pull request merged**